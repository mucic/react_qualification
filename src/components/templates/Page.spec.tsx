import React from "react";
import renderer from "react-test-renderer";

import Page from "./Page.component";

test("Page renders as expected", () => {
  const component = renderer.create(
    <Page>
        <p>Test page content</p>
    </Page>
  );

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});