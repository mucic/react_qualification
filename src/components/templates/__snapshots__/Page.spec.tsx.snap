// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders as expected 1`] = `
<div
  className="sc-AxjAm pkUjh"
>
  <p>
    Test page content
  </p>
</div>
`;
